rest/server.go

```diff
@@ -541,7 +543,8 @@ func (server *HTTPServer) NewMux() http.Handler {
 	httpHandler = xhttp.NewRequestLogger(httpHandler, server.Name(), serverExtraLogger, time.Millisecond, server.httpConfig.GetPackageLogger())
 
 	// metrics wrapper
-	httpHandler = xhttp.NewRequestMetrics(httpHandler)
+	extraMetrics := GetExtraMetrics(services)
+	httpHandler = xhttp.NewRequestWithExtraMetrics(httpHandler, extraMetrics)
```

Comment:
`httpHandler` is for single HTTP request. And now for each request, on specific service, all other service's Tags will be added?
I think you should not add this type of extension on Service level, but on the HTTP Server level, so all Services of HTTP Server install must provide the same Tags...
So, these Tags can be provided on the config of HTTP Server, rather than as an interface of the service.